src/elements/ScrollContainer/EndOfScrollWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+
+interface EndOfScrollWrapperProps {
+	onScrollToEnd: Function;
+	style: any;
+	children: any;
+}
+
+export default function EndOfScrollWrapper({
+	onScrollToEnd,
+	style,
+	children,
+}: EndOfScrollWrapperProps) {
+	const isCloseToBottom = ({ layoutMeasurement, contentOffset, contentSize }) => {
+		const paddingToBottom = 20;
+		return layoutMeasurement.height + contentOffset.y >= contentSize.height - paddingToBottom;
```

Comment:
nifty!

Reply:
Soooo... This one is kind of weird. I'm not sure why my commit history includes @CMChemist 's commits.